pandas/io/json/_normalize.py

```diff
@@ -490,6 +490,8 @@ def _pull_records(js: dict[str, Any], spec: list | str) -> list:
     meta_keys = [sep.join(val) for val in _meta]
 
     def _recursive_extract(data, path, seen_meta, level: int = 0) -> None:
+        if data is None:
```

Comment:
Isn't this generally an issue if `data` is not list like?

Reply:
While running the tests I noticed while passing scalar values must be done in a list as is the expected behavior. But if I have a row of data in my `json` and one of the field values is `null` then the `normalize()` fails by throwing an exception when I instead expect it to ignore the entire row, which this change should do.

Reply:
Could you add a test where a nested json value is a scalar like 2 as assert the result?

Reply:
Do you mean something like this as my expected value?
```
expected = DataFrame(
            {
                "nested_field_id": 1,
            }
        )
```

Reply:
hi all, what is the status on this one? Any idea when it could be merged? (original request: https://github.com/pandas-dev/pandas/issues/53719)

Reply:
@mroeschke Hi. In regard to the previous comment by me, is that what is required in the tests?

Reply:
A test where the input has something like `"nested_field": [{"nested_field_id": [1]}],`